src/Runner.Listener/Runner.cs

```diff
@@ -469,7 +469,8 @@ private void PrintUsage(CommandSettings command)
  --runnergroup string   Name of the runner group to add this runner to (defaults to the default runner group)
  --labels string        Extra labels in addition to the default: 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'
  --work string          Relative runner work directory (default {Constants.Path.WorkDirectory})
- --replace              Replace any existing runner with the same name (default false)");
+ --replace              Replace any existing runner with the same name (default false)
+ --ephemeral            Configure the runner only take one job and let service un-configured the runner after the job finish (default false)");
```

Comment:
👋 Small suggestion here:

```suggestion
 --ephemeral            Configure the runner to only take one job and then let the service un-configure the runner after the job finishes (default false)");
```

Reply:
Thanks, @martin389!